tests/test_modeling_utils.py

```diff
@@ -831,6 +832,20 @@ def test_from_pretrained_disk_offload_derived_to_base_model(self):
             outputs2 = new_model_with_offload(inputs)
             self.assertTrue(torch.allclose(outputs1[0].cpu(), outputs2[0].cpu()))
 
+    @slow
+    @require_torch
+    def test_from_pretrained_non_contiguous_checkpoint(self):
+        # See: https://github.com/huggingface/transformers/pull/28414
+        model = OwlViTForObjectDetection.from_pretrained("google/owlvit-base-patch16")
```

Comment:
It is possible to tests with a tiny model e.g. hf-internal-testing/tiny-random-owlvit? i.e. does this test with the tiny model fail on main?

Reply:
@amyeroberts sorry just added a comment. The tiny models have contiguous weights in the checkpoint so I can not reproduce the issue with the tiny models.

Reply:
Is it possible to create a tiny model with this property? This model is at least 100x larger than the tiny models.

Reply:
sure